src/public/index.html

```diff
@@ -51,7 +68,7 @@
                         <input class="form-control input-lg" type="email" name="email" id="email" required>
                     </div>
                     <div class="form-group">
-                        <label for="phone">Phone number</label>
+                        <label for="phone">Phone number, or text "____" to ____ to connect to us.</label>
```

Comment:
Maybe we could show this larger, above the form, instead of including it in this label. Once someone has started filling out the form, we don't want to tell them to connect with us some other way.